lib/functions.sh

```diff
@@ -3,12 +3,28 @@
 detectCompose() {
 	if [[ "$(docker compose version 2>/dev/null)" == *"Docker Compose version"* ]]; then
 		COMPOSE="docker compose"
-	else
+	e
```

Comment:
Even if this works, please don't do that

Reply:
Fixed. I don't recall ever changing this var.